openqa-post-similarity

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/python
+
+import os
+import Levenshtein
+from http import client
+from openqa_client.client import OpenQA_Client
+import json
+import argparse
+import logging
+from tqdm import tqdm
+import sys
+
+"""
+This script scans all autoinst-log.txt files in the testresults directory searching
+for error messages. Then it computes the Levenshtein distance and posts the
+results as openQA comments.
+"""
+DEBUG = True
+BASE_DIR = os.getenv("OPENQA_BASEDIR", "/var/lib/")
+if DEBUG:
+    TEST_DIR = BASE_DIR + "/repos/scripts/testresults/"
+else:
+    TEST_DIR = BASE_DIR + "/openqa/testresults/"
```

Comment:
As you already use argparse I suggest a command line option for this as well

Reply:
I have added an argument and set /var/lib/openqa/testresults/ as the default value